src/components/Welcome.js

```diff
@@ -0,0 +1,10 @@
+import React, { Component } from 'react'
```

Comment:
Even though this is a tiny component, it really should have a test since you are passing in a string and displaying it as a title.